test/e2e-common.sh

```diff
@@ -23,11 +23,23 @@
 
 # Helper functions for E2E tests.
 
+# Check if we have a specific RELEASE_YAML global environment variable to use
+# instead of detecting the latest released one from tektoncd/pipeline releases
+RELEASE_YAML=${RELEASE_YAML:-}
```

Comment:
Nice! Would you mind adding an example in the test README on how to use a specific release?

Reply:
Follow up PR should be fine for this :smile: